src/Chainweb/Miner/Config.hs

```diff
@@ -39,7 +42,7 @@ makeLenses ''MinerCount
 data MinerConfig = MinerConfig
     { _configTestMiners :: !MinerCount
     , _configMinerInfo :: !MinerInfo
-    , _configRemoteMiners :: ![HostAddress]
+    , _configRemoteMiners :: !(Set HostAddress)
```

Comment:
To ensure uniqueness from the outset. There is never a reason to target the same miner twice.